### ISSUE TITLE
Feature request: TLS Transport with valid certificates - DPI circumvention

### ISSUE THREAD

**Issue by iShift** [yggdrasil-network/yggdrasil-go#969]
Hi, for DPI circumvention it would be nice if Yggdrasil can use TLS certificates, or even communicate to Lets encrypt (or another ACME valid service) and acquire certificates.

**Comment by jhaar** [comment]
I agree. I have perfectly good LetsEncrypt cert I'd like to use. Currently the dynamically generated self-signed cert is being flagged as "TOR" by some Layer7 firewalls (like Fortinet), leading to yggdrasil being blocked. Having a legit cert (or at least a more legit looking cert) would most likely solve such issues

Jason
